### PR TITLE
feat: Update to website-3d-cell-viewer 2.9.0

### PR DESCRIPTION
**Pull request by toloudis**
*Edited by Peyton*

Viewer has a new way of passing array data in. This update simply adapts to use latest viewer version.

## Changelog
- Updates website-3d-cell-viewer to 2.9.0
- Rebuilds `yarn.lock` to fix a bug caused by conflicting versions of `d3-selection`
- Deletes old less and postCSS styling configuration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +29,7 @@
         "url": "https://github.com/allen-cell-animated/nbvv.git"
     },
     "scripts": {
-        "build": "jlpm build:styles && jlpm build:lib && jlpm build:labextension:dev",
-        "less": "lessc --js --math=always style/style.less style/style.css",
-        "postcss": "postcss style/style.css --config postcss.config.js -o style/style_out.css",
-        "build:styles": "jlpm less && jlpm postcss",
```
Comment (ShrimpCryptid) [inline package.json:35]: Deleted CSS transforming, since Ant no longer uses less for style management.

File: package.json [hunk 2]
```diff
@@ -101,6 +97,9 @@
         "typescript": "~5.0.2",
         "yjs": "^13.5.0"
     },
+    "resolutions": {
+        "d3-selection": "1.3.0"
+    },
```
Comment (ShrimpCryptid) [inline package.json:102]: Fixes a bug caused by two versions of d3-selection being included in the project, v1.1.1 and v1.3.0. This caused the widget to crash if the volume settings were opened.

File: yarn.lock
Comment (ShrimpCryptid) [inline yarn.lock:1]: Ended up deleting and regenerating yarn.lock to fix the aforementioned bug with `d3-selection`.

File: package.json (hunk 2 above)
Comment (toloudis) [reply package.json:101]: What was bringing 1.1 and what was bringing 1.3?

**Comment by toloudis** [comment]
Approved by me!  Assuming all the demo notebooks work :)
Before releasing, we might need a readme change to say it's only tested and built for jupyterlab and not the old jupyter notebook?

File: package.json (hunk 2 above)
Comment (ShrimpCryptid) [reply package.json:101]: It looks like `d3-transition` and `d3-zoom` required `d3-selection@npm: ^1.1.0`, both of which are dependencies of `d3`. I'm seeing two types of imports for d3-selection:
```
dependencies:
    d3-selection: 1
```
```
dependencies:
    d3-selection: ^1.1.0
```

**Review by ShrimpCryptid — APPROVED** [review]
Approving on behalf of Dan because I got requested as a reviewer even though I'm submitting this 😅😅

**Review by blairlyons — APPROVED** [review]

**Pull request merged**